ppx/src/reactjs_jsx_ppx.ml

```diff
@@ -122,6 +122,15 @@ let unerasableIgnore loc =
     attr_loc = loc;
   }
 
+(* [merlinHide] tells merlin to not look at a node, or at any of its
+   descendants. *)
+let merlinHide =
+  {
```

Comment:
this is only ever used as a list with a single element. we could allocate that list just once here.

Reply:
good point, fixed in https://github.com/reasonml/reason-react/pull/748/commits/ea0292763721fc82e95d16562bf4325af8c71bbb.